Base.rte/AI/HumanBehaviors.lua

```diff
@@ -1356,15 +1365,19 @@ function HumanBehaviors.GoToWpt(AI, Owner, Abort)
 								
 								if digState == AHuman.NOTDIGGING and AI.deviceState ~= AHuman.DIGGING then
 									-- if our path isn't blocked enough to dig, but the headroom is too little, start crawling to get through
-									local Heading = SceneMan:ShortestDistance(Owner.Pos, Waypoint.Pos, false):SetMagnitude(Owner.Height*0.5)
+									local heading = SceneMan:ShortestDistance(Owner.Pos, Waypoint.Pos, false):SetMagnitude(Owner.Height*0.5)
 									
-									-- don't crawl if it's too steep, climb then instead
-									if math.abs(Heading.X) > math.abs(Heading.Y) and Owner.Head and Owner.Head:IsAttached() then
-										local TopHeadPos = Owner.Head.Pos - Vector(0, Owner.Head.Radius*0.7)
+									local angleRadians = math.abs(math.atan2(-(heading.X * heading.Y), heading.X * heading.X));
+									local angleDegrees = angleRadians * (180 / math.pi);
+
+									-- We only crawl it it's quite flat, otherwise climb
+									local crawlThreshold = 30
```

Comment:
Might be good to specify in the name that it's a threshold angle or threshold degrees or something, so it's more obvious.
Also, this file isn't great for it cause it's mostly old stuff before our time, but we prefer to have `;`s at the end of our lua lines

Reply:
I automatically wrote ; at the end of every line and went back and deleted it afterwards for the sake of consistency. In my opinion, it's better to stay consistent on this until somebody (perhaps I) gets the change to change the entire file.

On the variable name - good point, I'll do that

Reply:
I have 46 lua files with the new magnitude junk that I forgot to push, so once this is merged I can do this and all the garbage trailing whitespace in a couple of hours if you want.

Reply:
That sounds good to me!

Actually I was just mentioned to Gacyr how it's probably better not to have semicolons here (I know, right?!). It's not semantically all that meaningful in Lua, and it's much easier to find-replace to remove semicolons than it is to add them. But if you're doing these changed, I certainly wouldn't object - consistency is what's important to me.